src/openforms/forms/models/form_variable.py

```diff
@@ -0,0 +1,105 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from ..constants import (
+    FormVariablesDataTypes,
+    FormVariablesInitialValues,
+    FormVariablesSources,
+)
+from .form import Form
+from .form_definition import FormDefinition
+
+
+def get_today():
+    return timezone.now().isoformat()
```

Comment:
confusing - `today` implies a `date`, but `timezone.now()` returns a `datetime`. Also, please add type hints :-)

Reply:
Yes :sweat_smile: I made it a datetime to avoid the mess of timezones and dates... maybe instead of 'today' i should call it 'now' or something like this?